### DISCUSSION TITLE
Can't get student pack

### DISCUSSION THREAD

**Discussion by kozhinae** [community/community#77704] [GitHub Education]
### Hi everyone,

I was approved on 16 November. Today is 02 December and I still can't get student developer pack. How can I solve this problem?
![image](https://github.com/community/community/assets/89837526/2a75efc5-b7f7-46e1-9fe8-61c8bd0f8585)
![image](https://github.com/community/community/assets/89837526/522031af-ed57-4271-bc7f-dc7d78b2ba0c)

**Comment by mesh-dell** [comment]
1. Check your eligibility: Ensure you meet all the eligibility criteria for the Student Developer Pack. Verify that you're an actively enrolled student at a qualified educational institution and have a valid student email address.

2. Review your application: Double-check your application information to ensure there are no errors or inconsistencies. Make sure you provided the correct student email address and that the email address is associated with your GitHub account.

3. Wait for processing: It may take some time for GitHub to process your application and grant you access to the Student Developer Pack benefits. Typically, it takes 1-2 business days, but during peak periods, it could take longer.

4. Contact GitHub support: If it's been more than 15 business days since your approval, and you still haven't received the benefits, contact GitHub support for assistance. They can help you investigate the issue and expedite the process.

**Comment by dakshithadissanayaka** [comment]
Based on the provided information, you were approved for the student developer pack on November 16, 2023. Today is December 2, 2023, and you are still unable to claim the pack.

There are a few possible reasons for this:

* **Processing delay:** The student developer pack approval process may take some time to complete, especially if there is a high volume of applications. In this case, you may need to wait a few more days for the pack to become available.
* **Technical issue:** There may be a technical issue preventing you from claiming the pack. This could be a problem with the GitHub system, your account, or the pack itself.
* **Eligibility issue:** It is also possible that you are no longer eligible for the student developer pack. This could be due to a change in your student status, your GitHub account, or the pack itself.

To resolve this issue, you can try the following steps:

1. **Check your eligibility:** Make sure that you are still eligible for the student developer pack. You can check your eligibility on the GitHub website.
2. **Try to claim the pack again:** Once you have confirmed that you are eligible, try to claim the pack again. If you are still unable to claim the pack, contact GitHub support for assistance.
3. **Contact GitHub support:** If you have tried all of the above steps and you are still unable to claim the pack, contact GitHub support for assistance. They can investigate the issue further and provide more specific troubleshooting steps or identify any underlying issues.

When contacting GitHub support, be sure to provide them with as much information as possible about the issue, including the following:

* Your GitHub username
* The date you were approved for the student developer pack
* The error message you are receiving when trying to claim the pack (if any)
* Any other relevant information

By providing GitHub support with this information, they will be better able to assist you in resolving the issue.
